docs/source/usage/plugins/transitionRadiation.rst

```diff
@@ -216,7 +216,8 @@ For a specific (charged) species ``<species>`` e.g. ``e``, the radiation can be
 ========================================== ==============================================================================================================================
 Command line option                        Description
 ========================================== ==============================================================================================================================
-``--<species>_transitionRadiation.period`` Gives the number of time steps between which the radiation should be calculated.
+``--<species>_transRad.period``            Gives the number of time steps between which the radiation should be calculated.
+``--<species>_transRad.omegaList```        Currently deprecated
```

Comment:
Why are you still listing this command line option, even though it does not appear in [TransitionRadiation.hpp](https://github.com/ComputationalRadiationPhysics/picongpu/blob/f495273b3a398fbe45ca08feb72a74d15b7b18be/include/picongpu/plugins/transitionRadiation/TransitionRadiation.hpp#L176)? Is it the `omegaList` still used somewhere else?

Reply:
@finnolec Great catch - I just copy pasted the wrong documentation 😞